packages/accounts/src/msca/plugins/multisig/extension.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  AccountNotFoundError,
+  type GetAccountParameter,
+  type IsUndefined,
+  type SmartContractAccount,
+} from "@alchemy/aa-core";
+import { type Address, type Chain, type Client, type Transport } from "viem";
+import type { GetPluginAddressParameter } from "../types.js";
+import {
+  MultisigPlugin,
+  multisigPluginActions as multisigPluginActions_,
+  type MultisigPluginActions as MultisigPluginActions_,
+} from "./plugin.js";
+
+export type MultisigPluginActions<
+  TAccount extends SmartContractAccount | undefined =
+    | SmartContractAccount
+    | undefined
+> = MultisigPluginActions_<TAccount> & {
+  readOwners: (
+    params: GetPluginAddressParameter & GetAccountParameter<TAccount>
+  ) => Promise<ReadonlyArray<Address>>;
+
+  isOwnerOf: (
+    params: { address: Address } & GetPluginAddressParameter &
+      GetAccountParameter<TAccount>
+  ) => Promise<boolean>;
+
+  getThreshold: (
+    params: GetPluginAddressParameter & GetAccountParameter<TAccount>
+  ) => Promise<bigint>;
+} & (IsUndefined<TAccount> extends false
+    ? {
+        readOwners: (
+          params?: GetPluginAddressParameter & GetAccountParameter<TAccount>
+        ) => Promise<ReadonlyArray<Address>>;
+      }
+    : {});
+
+export const multisigPluginActions: <
+  TTransport extends Transport = Transport,
+  TChain extends Chain | undefined = Chain | undefined,
+  TAccount extends SmartContractAccount | undefined =
+    | SmartContractAccount
+    | undefined
+>(
+  client: Client<TTransport, TChain, TAccount>
+) => MultisigPluginActions<TAccount> = <
+  TTransport extends Transport = Transport,
+  TChain extends Chain | undefined = Chain | undefined,
+  TAccount extends SmartContractAccount | undefined =
+    | SmartContractAccount
+    | undefined
+>(
+  client: Client<TTransport, TChain, TAccount>
+) => ({
+  ...multisigPluginActions_(client),
+  async readOwners(
+    args: GetPluginAddressParameter & GetAccountParameter<TAccount>
+  ) {
+    const account = args?.account ?? client.account;
+    if (!account) {
```

Comment:
```suggestion
    if (account == null) {
```

Reply:
ill have to change the one in multiowner also to be consistent, is that fine?